lib/rex/post/postgresql/ui/console.rb

```diff
@@ -113,34 +60,18 @@ def log_error(msg)
             dlog("Call stack:\n#{$@.join("\n")}", 'postgresql')
           end
 
-          #
-          # Interacts with the supplied client.
-          #
-          def interact_with_client(client_dispatcher: nil)
-            return unless client_dispatcher
-
-            client.extend(InteractiveSqlClient) unless (client.kind_of?(InteractiveSqlClient) == true)
-            client.on_command_proc = self.on_command_proc if self.on_command_proc
-            client.on_print_proc   = self.on_print_proc if self.on_print_proc
-            client.on_log_proc = method(:log_output) if self.respond_to?(:log_output, true)
-            client.client_dispatcher = client_dispatcher
-
-            client.interact(input, output)
-            client.reset_ui
-          end
-
           # @return [Msf::Sessions::PostgreSQL]
           attr_reader :session
 
           # @return [PostgreSQL::Client]
           attr_reader :client # :nodoc:
 
           # @return [String]
-          attr_accessor :cwd
+          attr_accessor :database_name
 
           def format_prompt(val)
-            cwd ||= client.params['database']
-            prompt = "%undPostgreSQL @ #{client.conn.peerinfo} (#{cwd})%clr > "
+            db_name ||= client.params['database']
```

Comment:
Might be worth checking if this is happening in the way you expect it is working 👀 

I believe if we want to assign to database_name we need `self.database_name = ...` - but it looks like the names aren't the same either, so it's maybe dead code? 🤔